src/couch_pse_tests/src/cpse_util.erl

```diff
@@ -183,16 +183,24 @@ uuid() ->
     couch_uuids:random().
 
 
-assert_db_props(DbName, Props) when is_binary(DbName) ->
+assert_db_props(DbName, Props, Module, Line) when is_binary(DbName) ->
```

Comment:
Nitpick (kind of): I would put `Module` and `Line` to be first arguments. The reason is pure esthetics (very subjective). We expect `Props` to be long so it is better to have it as the very last argument. Compare:

```
assert_db_props(?MODULE, ?LINE, DbName, [
         {doc_count, 1},
         {doc_count, 1},
         {del_doc_count, 0},
         {del_doc_count, 0},
         {update_seq, 2},
         {update_seq, 2},
         {purge_seq, 0},
         {purge_seq, 0},
         {purge_infos, []}
]).
```

with 

```
assert_db_props(DbName, [
         {doc_count, 1},
         {doc_count, 1},
         {del_doc_count, 0},
         {del_doc_count, 0},
         {update_seq, 2},
         {update_seq, 2},
         {purge_seq, 0},
         {purge_seq, 0},
         {purge_infos, []}
], ?MODULE, ?LINE).

```

On the other hand most of the functions in erlang stdlib don't have this convention. For example all functions in `lists` module has long function definition as the first argument. I guess your version is less surprising to erlang developers. Therefore I propose to keep yours. Unless you think it would be better to change the order.

Reply:
Another option is to define macro (somewhere) as:
```
-define(assert_db_props(DbName, Props), 
      cpse_util:assert_db_props(DbName, Props, ?MODULE, ?LINE)).
```

Then you don't need to pass additional arguments at all:
```
?assert_db_props(DbName, [
         {doc_count, 1},
         {doc_count, 1},
         {del_doc_count, 0},
         {del_doc_count, 0},
         {update_seq, 2},
         {update_seq, 2},
         {purge_seq, 0},
         {purge_seq, 0},
         {purge_infos, []}
]).
```

Reply:
As I said not a big deal. Feel free to ignore.

Reply:
Move `?MODULE` and `?LINE` to the beginning of `assert_db_props/4` using commit 43f3183d275a4fae6f7ba5332ddca0a498315ec9